great_expectations/checkpoint/actions.py

```diff
@@ -1044,19 +1080,8 @@ def _run(  # type: ignore[override] # signature does not match parent
         checkpoint_identifier=None,
         **kwargs,
     ):
-        suite_name: str = validation_result_suite.meta["expectation_suite_name"]
```

Comment:
I'm confused on this change. This is a separate action, right?

Reply:
Yup but `mypy` kept flagging it as an issue

Reply:
Probably because of the change to `meta` below